ajax/libs/react-datetime/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "react-datetime",
+  "version": "2.8.9",
+  "filename": "dist/react-datetime.min.js",
+  "description": "A lightweight but complete datetime picker React.js component.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/YouCanBookMe/react-datetime.git"
+  },
+  "keywords": [
+    "react",
+    "react-component",
+    "datepicker",
+    "timepicker",
+    "datetimepicker",
+    "datetime"
+  ],
+  "author": "Javier Marquez",
+  "license": "MIT",
+  "npmName": "react-datetime",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "dist/**/*",
+        "css/**/*",
+        "src/**/*"
```

Comment:
If `dist`, `css`, `src` separate three rules, these files will all in version folder.
But I think it's not a good method because I saw some library in cdnjs is distinguished between folders.
like https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/jquery-ui-bootstrap/0.5pre